package.json

```diff
@@ -96,10 +97,10 @@
     "generate-size": "browserify . -p tinyify -s micromark -o micromark.min.js; gzip-size micromark.min.js; gzip-size --raw micromark.min.js",
     "generate": "npm run generate-constant-typings && npm run generate-expressions && npm run generate-dist && npm run generate-size",
     "format": "remark . -qfo && prettier . -w --loglevel warn && xo --fix",
-    "pretest-fuzz": "patch-package --patch-dir script/patches; node script/generate-fixtures",
-    "test-fuzz": "timeout 15m jsfuzz test/fuzz.js test/fixtures --rss-limit-mb 300 --timeout 300",
-    "test-api": "node test",
-    "test-coverage": "nyc --reporter lcov tape test/index.js",
+    "pretest-fuzz": "patch-package --patch-dir script/patches; node script/generate-fixtures.mjs",
+    "test-fuzz": "timeout 15m NODE_OPTIONS='-r esm' jsfuzz test/fuzz.js test/fixtures --rss-limit-mb 300 --timeout 300",
+    "test-api": "node test/index.mjs",
+    "test-coverage": "nyc --reporter lcov node test/index.mjs",
     "test-types": "dtslint .",
     "test": "npm run generate && npm run format && npm run test-coverage && npm run test-types"
   },
```

Comment:
I did not have luck setting:

```
    "extensions": [
      ".cjs",
      ".mjs"
    ],
```

in `xo` below, but it did work passing them directly to `xo` in the `format` script above.

The last section on the xo overrides deal with the tests. I think it should be:

```
      {
        "files": [
          "test/**/*.{js,mjs,cjs}"
        ],
        "rules": {
          "import/extensions": "off",
          "import/no-unassigned-import": "off",
          "import/no-useless-path-segments": "off",
          "unicorn/import-index": "off"
        }
      }
```

Reply:
It's probably the latest eslint behaviour